pallets/route-executor/src/lib.rs

```diff
@@ -54,6 +54,13 @@ pub use pallet::*;
 
 pub const MAX_NUMBER_OF_TRADES: u32 = 5;
 
+#[derive(Debug, Encode, Decode, Copy, Clone, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+pub enum SkipEdState {
+	SkipEdLock,
```

Comment:
i think you can remove SkipEd prefix from the enum names

Reply:
I think @mrq1911 ment:
`SkipEdState {
	Lock,
	LockAndUnlock,
	Unlock,
}`

"ed" is still redundant in the usage of this enum
I would seggest:
`
SkipED {
        Lock,
	LockAndUnlock,
	Unlock,
}
`
"state"/"handling" in name doesn't add much in my opinion